plotly/figure_factory/_dendrogram.py

```diff
@@ -88,20 +89,19 @@ def create_dendrogram(X, orientation="bottom", labels=None,
 
     dendrogram = _Dendrogram(X, orientation, labels, colorscale,
                              distfun=distfun, linkagefun=linkagefun,
-                             hovertext=hovertext)
+                             hovertext=hovertext, color_threshold=color_threshold)
 
-    return graph_objs.Figure(data=dendrogram.data,
-                             layout=dendrogram.layout)
+    return graph_objs.Figure(data=dendrogram.data, layout=dendrogram.layout)
 
 
 class _Dendrogram(object):
     """Refer to FigureFactory.create_dendrogram() for docstring."""
 
     def __init__(self, X, orientation='bottom', labels=None, colorscale=None,
-                 width=np.inf, height=np.inf, xaxis='xaxis', yaxis='yaxis',
+                 width="100%", height="100%", xaxis='xaxis', yaxis='yaxis',
```

Comment:
"100%" is not a valid height/width and it looks like this is causing your test failure.

Where you trying to change some behavior here? I'm not sure why the old value was `inf`. I think the default should probably be just `None`, which will let the rending approach (`plot`, `iplot`, or `FigureWidget`) decide how to size the figure.